src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -923,6 +924,69 @@ public static void prepareEnv(String projectPath, String projectName) {
                 "prepareEnv. Exit. ProjectName: " + projectName);
     }
 
+    /**
+     * Tests that when a Liberty run configuration is removed, any custom start parameters
+     * linked to the Liberty module are also cleared.
+     */
+    @Test
+    @Video
+    public void testCustomStartParametersClearedOnConfigRemoval() {
+        String testName = "testCustomStartParametersClearedOnConfigRemoval";
+        String absoluteWLPPath = Paths.get(getProjectsDirPath(), getSmMPProjectName(), getWLPInstallPath()).toString();
+
+        // Remove all existing configurations for a clean state.
+        UIBotTestUtils.deleteLibertyRunConfigurations(remoteRobot);
+
+        // Trigger the start with parameters configuration dialog.
+        UIBotTestUtils.runLibertyActionFromLTWDropDownMenu(remoteRobot, "Start...", false, 3);
+
+        // Run the configuration dialog.
+        UIBotTestUtils.runStartParamsConfigDialog(remoteRobot, getStartParamsDebugPort());
+
+        try {
+            // Validate that the project started.
+            TestUtils.validateProjectStarted(testName, getSmMpProjResURI(), getSmMpProjPort(), getSmMPProjOutput(), absoluteWLPPath, false);
+
+            // Expand the Project tree and open server.env file
+            UIBotTestUtils.openFile(remoteRobot, getSmMPProjectName(), "server.env", getServerEnvPath());
```

Comment:
What is the reason we need to open the `server.env` file in the editor?

Reply:
We need to read the server.env file and check for the debug port value.

Reply:
We are using the Java Files library to read the `server.env` file in `checkDebugPort()`. There is no connection between using that library and what is open in the IntelliJ editor.

Reply:
Removed the openFile method. We are only reading the server.env file instead of opening it.